llama-index-core/llama_index/core/workflow/workflow.py

```diff
@@ -116,6 +127,8 @@ async def _task(
                     args = []
                     if config.pass_context:
                         args.append(self.get_context(name))
+                    for service in config.services:
```

Comment:
It seems a bit awkward to pile on more positional arguments. I would prefer this be handled by a keyword argument(s).

Reply:
I'm not sure, this would mean having a convention over parameter names (e.g., events have to be named `event`, the context instance `context`, and so on and so forth).

Reply:
Took a stab at it in a local branch, it's doable but I need way more introspection than what I'm doing here, let's track this in a separate issue/PR

Reply:
Moved to keyword arguments in the end 👍